docs/project-setup/filesystem-sync.md

```diff
@@ -48,12 +48,11 @@ The file synchronization is managed by the rig cli via `rig project sync:start`
 
 The name of the sync volume can be set in a variety of ways. It is determined using the following precedence:
 
-1. Argument provided to the `project sync:start` command
-2. Specified in the `sync` -> `volume-name` property of your [Outrigger Project Configuration](./project-configuration.md)
-3. In your `docker-compose.yml` file, specified as an `external` volume with a name of the pattern `*-sync`
-4. If not specified anywhere else, it will use the name of the current project folder with `-sync` appended
+1. Specified in the `sync` -> `volume-name` property of your [Outrigger Project Configuration](./project-configuration.md)
+2. In your `docker-compose.yml` file, specified as an `external` volume with a name of the pattern `*-sync`
+3. If not specified anywhere else, it will use the name of the current project folder with `-sync` appended
 
-We recommend using approach #2 or #3 to ensure consistency across all environments. Approach #3 is most often used as it requires the most minimal configuration additions. If you use approach #2, make sure a volume of the same name is also defined as part of your docker or docker-compose configuration. Approach #2 is useful when a need exists to restrict which files are synced or the name of the volume can not follow the `*-sync` naming pattern. An example of limiting the synced files can be found in the sample in the [Outrigger Project Configuration](./project-configuration.md) documentation.
+We recommend using approach #1 or #2 to ensure consistency across all environments. Approach #2 is most often used as it requires the most minimal configuration additions. If you use approach #1, make sure a volume of the same name is also defined as part of your docker or docker-compose configuration. Approach #1 is useful when a need exists to restrict which files are synced or the name of the volume can not follow the `*-sync` naming pattern. An example of limiting the synced files can be found in the sample in the [Outrigger Project Configuration](./project-configuration.md) documentation.
```

Comment:
I realize the text is the same, but this took me a couple reads to follow: is approach #1 a requirement to add other unison configurations like ignores? Or could there just be that section in the outrigger.yml without specifying the volume name?